babel.config.js

```diff
@@ -1,9 +1,9 @@
-const defaultPresets = ['@babel/preset-react', '@babel/preset-env'];
+const defaultPresets = ['@babel/preset-react', '@babel/preset-env', '@babel/preset-flow'];
```

Comment:
I needed babel to transpile the flow annotations so I added the babel preset-flow

Reply:
Makes sense to me.